content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -1,22 +1,19 @@
 ---
 reviewers:
 - jessfraz
-title: Inject Information into Pods Using a PodPreset
+title: Inject Information into Pods Using a Pod Preset
 content_template: templates/task
 weight: 60
 ---
 
 {{% capture overview %}}
 
-You can use a `PodPreset` object to inject information like secrets, volume
-mounts, and environment variables etc into pods at creation time.
-This task shows some examples on using the `PodPreset` resource.
+This page shows how to use Pod presets to inject information like secrets, volume mounts, and environment variables into Pods at creation time.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-Get an overview of PodPresets at
```

Comment:
```suggestion
If you haven't used a PodPreset already, take a quick look at
```
?
(not sure about “quick”; what I want to say is that since this is a tutorial-style task it's OK for the reader to glance through and then come back even if they don't understand everything they have seen).

Also, do readers need to know what Pods are?

Reply:
I was scanning through the other pages in the `tasks` section of the documentation and most of them do not contain any links to the concepts they cover. So I was thinking about removing that sentence altogether.

Reply:
That works too!